ejb3/src/main/java/org/jboss/as/ejb3/subsystem/EJB3SubsystemRootResourceDefinition.java

```diff
@@ -73,7 +73,8 @@
                     .setAllowExpression(true).build();
     static final SimpleAttributeDefinition DEFAULT_RESOURCE_ADAPTER_NAME =
             new SimpleAttributeDefinitionBuilder(EJB3SubsystemModel.DEFAULT_RESOURCE_ADAPTER_NAME, ModelType.STRING, true)
-                    .setDefaultValue(new ModelNode().set("hornetq-ra"))
+                    // FIXME this should not be hard-coded
+                    .setDefaultValue(new ModelNode().set("activemq-ra"))
```

Comment:
I changed this value without much thought (and none about migration).
Now you make me think of it, there might be some issues with that attribute.

First, I'm not sure that this default value is correct at all (even the
previous value).
The ejb3 subsystem redefines the value of the attribute in its XML template:

```
<resource-adapter-ref
resource-adapter-name="${ejb.resource-adapter-name:activemq-ra.rar}"/>
```
The value that works is `activemq-ra.rar`. I'll check if `activemq-ra` is
correct.

Its value corresponds to a resource adapter that can come from many places
(the messaging subsystem, the resource-adapter subsystem or the new
messaging-activemq subsystem).
I remember adding an expression to this attribute to be able to use the
generic JMS RA as the default MDB RA.

I wonder if this attribute should have a default value at all. Since we
explicitly redefine it in our profile, it looks like we think this must be
an explicit configuration.

@bstansberry  What do you think about removing its default value? I'll
still have to transform it but it seems more correct to make it an explicit
configuration instead of relying on some magic value from another subsystem.